package.json

```diff
@@ -109,9 +109,9 @@
     "ts-dedent": "^2.0.0"
   },
   "peerDependencies": {
-    "@storybook/core-common": "^6.4.0 || >= 6.5.0-alpha.1",
-    "@storybook/csf-tools": "^6.4.0 || >= 6.5.0-alpha.1",
-    "@storybook/store": "^6.4.0 || >= 6.5.0-alpha.1",
+    "@storybook/core-common": ">= 6.5.0-rc.0",
```

Comment:
probably just `^6.5.0` now that it's live?

Reply:
done